improver_tests/set_up_test_cubes.py

```diff
@@ -60,48 +62,130 @@ def construct_xy_coords(ypoints, xpoints, spatial_grid):
             Number of grid points required along the x-axis
         spatial_grid (str):
             Specifier to produce either a "latlon" or "equalarea" grid
+        grid_spacing (int or float):
+            Grid resolution (degrees or metres).
+        domain_corner (list of int or float):
+            Bottom left corner of grid domain [y,x] (degrees or metres).
```

Comment:
Could you just add to this the default behaviour - ie if `domain_corner` is not provided, we create a grid centred on (0, 0).